src/main/java/com/zontik/groshiky/controllers/AccountController.java

```diff
@@ -17,12 +20,16 @@
     private final IAccountService accountService;
     private final IUserService userService;
     private final ITransactionService transactionService;
+    private final ModelMapper mapper;
```

Comment:
It's better to put model mapper into BaseController as protected variable and use everywhere